_config-sta.yml

```diff
@@ -0,0 +1,34 @@
+name: devit
+description: The DEVit conference
+
+permalink: /:categories/:title
+
+disqus: devitconf
+
+exclude: [
+  "node_modules",
+  "gulpfile.js",
+  "package.json",
+  ".*",
+  "CONTRIBUTING.md",
+  "LICENSE",
+  "README.md",
+  "_js"
+]
+
+og:
+  title: DEVit the 360° Web Development Conference
+  description: Find out the latest trends and techniques for Frontend, Backend, DevOps and Mobile! Meet all the cool people! We are 360° and proud of it!
+
+analytics:
+  google:
+    tracker: 'UA-41787746-8'
```

Comment:
Totally agree. I do not know how it will behave by removing it. We need a specific setup that uses `'cookieDomain': 'none'` as an option and brings the GA tag on a specific state. Some conditional logic on the template. Will have a look tomorrow.